package dio.me.desafiofinal.santander.application.services.exceptions.dto;

public record AlreadyExistsExceptionDTO(String model, String message) {
}
